packages/xs/xapi-idl.1.90.1/opam

```diff
@@ -10,10 +19,11 @@ remove: [
 depends: [ "uri"
            "re"
            "cmdliner"
-           "cohttp" {>= "0.15.2"}
+           "cohttp"
            "xmlm"
-           "rpc" {>= "1.9.4"}
+           "rpc" {>= "1.5.3"}
```

Comment:
Do we need this version constraint? The way xs-opam is constructed, there is only one version per component and either this works or it doesn't.